packages/in_app_purchase/in_app_purchase_storekit/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -214,6 +233,14 @@ class SKProductDiscountWrapper {
   /// and their units and duration do not have to be matched.
   final SKProductSubscriptionPeriodWrapper subscriptionPeriod;
 
+  /// A string used to uniquely identify a discount offer for a product.
```

Comment:
Can you add a comment explain what does it mean to have a "null" identifier and how should user handle it.

Reply:
It is nullable defined by iOS SDK. And this comment is copied from it's document.
My explanation may lead to confusion.

Reply:
Lets also add some explanation to the doc: 
```
/// A string used to uniquely identify a discount offer for a product.
///
/// You set up offers and their identifiers in App Store Connect.

```